lms/urls.py

```diff
@@ -666,6 +666,12 @@
         ),
         include('openedx.features.portfolio_project.urls'),
     ),
+
+    # Subscriptions
+    url(
```

Comment:
We should make `subscription` a [plugin](https://github.com/edx/edx-platform/blob/master/docs/guides/extension_points.rst#platform-features-integrating-python-code) 
https://github.com/edx/edx-django-utils/blob/master/edx_django_utils/plugins/README.rst